include/votca/csg/topology.h

```diff
@@ -271,19 +272,15 @@ class Topology {
       boxtype = autoDetectBoxType(box);
     }
 
-    if (_bc) {
-      delete (_bc);
-    }
-
     switch (boxtype) {
       case BoundaryCondition::typeTriclinic:
-        _bc = new TriclinicBox();
+        _bc = std::unique_ptr<BoundaryCondition>(new TriclinicBox());
```

Comment:
```suggestion
        _bc = std::unique_ptr<TriclinicBox>();
```

Reply:
lol this is broken.

Reply:
`std::make_unique<TriclinicBox>()`

sorry yesterday I was not at my best.